src/game/boe.main.cpp

```diff
@@ -250,6 +250,15 @@ void process_args(int argc, char* argv[]) {
 	}
 }
 
+void replay_next_action() {
+	auto next_action = pop_next_action();
+	std::string t = next_action->Value();
+	if (t == "load_party") {
+		decode_file(next_action->GetText(), "temp.exg");
+		load_party("temp.exg", univ);
```

Comment:
This seems a bit sloppy. Where is that file placed on disk? I would guess it ends up in the current working directory, which is probably right beside the executable, but that's a problem, because it's usually write-protected.

I'd say it should be put in the same place that temporary scenario files are written to when editing a packed scenario. I think that was something like a "temp" folder within the settings folder.